input/otlp/traces.go

```diff
@@ -812,6 +816,11 @@ func TranslateSpan(spanKind ptrace.SpanKind, attributes pcommon.Map, event *mode
 				// stacktrace is expected to be large thus un-truncated value is needed
 				event.Code.Stacktrace = v.Str()
 
+			// gen_ai.*
+			case "gen_ai.system":
```

Comment:
nitpick: I would suggest to move this to a constant.

Here the change would be:
```suggestion
			case attributeGenAiSystem:
```

and the constant should be added to the beginning of the file, below https://github.com/xrmx/apm-data/blob/64e886dbca1d5ae08ac23b6e7862942c4fc65859/input/otlp/traces.go#L66-L84

Reply:
oh, missed that, updated!